src/code/z_debug.c

```diff
@@ -14,17 +14,19 @@ typedef struct {
 
 RegEditor* gRegEditor;
 
-PrintTextBufferEntry sDebugPrintTextBuffer[22];
-s16 sDebugPrintTextBufferNumUsed = 0;
+PrintTextBufferEntry sDebugTextBuffer[22];
+
+s16 sDebugTextEntryCount = 0;
+
 Color_RGBA8 sDebugPrintTextColors[] = {
```

Comment:
"text" vs "print text" ?
"print text" would be -> master,
committing to "text" may be:
```suggestion
DebugTextBufferEntry sDebugTextBuffer[22];

s16 sDebugTextEntryCount = 0;

Color_RGBA8 sDebugTextColors[] = {
```

Reply:
yeah just missed these while removing masters "print"